Note DCID must be extracted from Client Initial (#380)
quicwg/ops-drafts: draft-ietf-quic-manageability.md

When the version has been identified as QUIC version 1, the packet type needs to
be verified as an Initial packet by checking that the third and fourth bits of
the header are both set to 0. Then the Destination Connection ID needs to be

> extracted from the Client Initial to calculate the Initial secret using the

Reviewer: ```suggestion
extracted from the packet. The Initial secret is calculated using the
```

"Client Initial" is not really a thing in this context (you don't really want to reuse that context here, I think).  As you are talking about packets in the abstract, I think that this is right.  The next paragraph covers the important gotcha.
